packages/web3-utils/src/internal.ts

```diff
@@ -0,0 +1,155 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/**
+ * Returns a `Boolean` on whether or not the a `String` starts with '0x'
+ * @param str the string input value
+ * @return a boolean if it is or is not hex prefixed
+ * @throws if the str input is not a string
+ */
+export function isHexPrefixed(str: string): boolean {
+	if (typeof str !== 'string') {
+		throw new Error(`[isHexPrefixed] input must be type 'string', received type ${typeof str}`);
+	}
+
+	return str.startsWith('0') && str[1] === 'x';
```

Comment:
```suggestion
	return str.startsWith('0x');
```
Don't you already have most of these methods in web3-utils and web3-validator?